.gitignore

```diff
@@ -0,0 +1,266 @@
+.directory
```

Comment:
why do we want this copied from main repo?

Reply:
Well, it has some nice "defaults" even if not everything in it will show up in the repo.

If you want... I can generate one directly around the web repo, but its liable to be even longer since I'll add in a bunch of specifics like `.directory` that are all platform specific.

Know a good site for generating them after all :)

Reply:
Generated repo specifc gitignore. Just dont complain about its length since you asked for it! :P